appveyor.yml

```diff
@@ -0,0 +1,68 @@
+environment:
+
+  # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the /E:ON and
+  # /V:ON options are not enabled in the batch script interpreter See:
+  # http://stackoverflow.com/a/13751649/163740
+  CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
+
+  # Workaround for https://github.com/conda/conda-build/issues/636
+  PYTHONIOENCODING: "UTF-8"
+
+  matrix:
+    - TARGET_ARCH: "x64"
+      CONDA_PY: "34"
+      PY_CONDITION: "python >=3.4,<3.5"
+    - TARGET_ARCH: "x64"
```

Comment:
Yes python 3.6 fails to build, at least with the conda env approach I have taken here, as there are a huge amount of missing python 3.6 conda dependencies. They are in the process of re-rendering current feedstocks to get python 3.6 packages out so it would be long until this can be added.